doc/nrf/gs_programming.rst

```diff
@@ -87,4 +87,82 @@ Complete the following steps to build |NCS| projects with SES after :ref:`instal
    	In a multi-image build, this allows you to debug the source code of your application only.
 
 If you get an error that a tool or command cannot be found, first make sure that the tool is installed.
-If it is installed, verify that its location is correct in the PATH variable or, if applicable, in the SES settings.
\ No newline at end of file
+If it is installed, verify that its location is correct in the PATH variable or, if applicable, in the SES settings.
+
+.. _gs_programming_cmd:
+
+Building with Command line
+**************************
+
+Complete the following steps to build |NCS| projects with command line after completing the :ref:`command line build environment setup for windows<build_environment_cli_win>` or the :ref:`command line build environment setup for linux<build_environment_cli_linux>` for windows and linux environments respectively.
+
+1. Connect the development board to your PC using a USB cable.
+
+#. Power on the development board.
+
+#. Open a terminal window.
+
+#. Go to the samples or applications directory depending on whether you are building a sample or an application:
+
+   .. code-block:: console
+
+      cd nrf/samples
+      cd nrf/applications
+
+   .. note::
+      All the samples corresponding to the nRF9160 DK board are located at :file:`samples/nrf9160`.
+
+#. Build the sample or application using the west command.
+   The development board is specified by the parameter ``board_name``, and the sample or application directory is specified by the parameter ``directory_name`` in the west command as follows:
+
+   .. code-block:: console
+
+      west build -b board_name directory_name
+
+   See `Board names <Board names_>`_ for more information on the development boards.
+   If an existing build directory has to be re-used for building another sample or application for another board, pass ``-p=auto`` to ``west build``.
+
+   After running the ``west build`` command, we can see that the build files are generated at ``build/zephyr``.
+   For more information on the contents of the build directory, see the *Build Directory Contents* section in the Zephyr documentation on :ref:`zephyr:build_an_application`.
```

Comment:
*Build Directory Contents* could be a hyperlink instead.

Reply:
Changed to a hyper link.